repository.py

```diff
@@ -850,7 +850,10 @@ def updateInstallProgress(m):
         rv = p.wait()
 
         if rv:
-            logger.log("DNF exited with %d" % rv)
+            if rv > 0:
+                logger.log("DNF exited with %d" % rv)
+            else:
+                logger.log("DNF killed with signal %d" % -rv)
```

Comment:
Is it worth decoding the signal name too?

```
logger.log("DNF killed with signal: %d" % (signal.strsignal(-rv))
```

Reply:
and `%s` instead of `%d`.